core/src/main/scala/matryoshka/instances/fixedpoint/package.scala

```diff
@@ -67,4 +67,18 @@ package object fixedpoint {
         case (_, _)                 => NilF()
       }
   }
+
+  /** Encodes a function that may diverge.
+    */
+  type Partial[A] = Nu[A \/ ?]
+  type ⊥[A] = Partial[A]
```

Comment:
Here’s an example of the usage: http://www.cse.chalmers.se/~nad/publications/danielsson-semantics-partiality-monad.pdf. It’s easier to see in LaTeX, where you can distinguish `⊥` (or even `⊥_{A}`) from `A_{⊥}` (the last being the notion I intended). I guess it also makes more sense in a total language, where `A_{⊥}` is the only form of divergence available.

So I guess the `⊥` alias makes more sense for `Nothing` in Scala. Ok, I fold 😉